extractor/src/main/scala/com/digitalasset/extractor/config/ConfigParser.scala

```diff
@@ -50,7 +50,7 @@ object ConfigParser {
   private val configParser: OptionParser[CliParams] =
     new scopt.OptionParser[CliParams]("extractor") {
 
-      override def showUsageOnError: Boolean = true
+      override def showUsageOnError: Option[Boolean] = Some(true)
```

Comment:
As a minor side note, this can be a `val`. But more importantly, do you know what this API change means? An `Option[Boolean]` sounds a bit weird.

Reply:
Very good question, I admit I hadn’t looked at it.

Previously the behavior was: If `showUsageOnError` is true, show usage on error, else call `showTryHelp`. Now it is the following:
If `showUsageOnError` is true show usage on error, if it is false call `showTryHelp`. If it is nothing treat it as true iff there is no help option, otherwise as false. So setting it to `Some(b)` appears to be the equivalent of `b` before.